Courses/Views/Screens/MatchedView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  MatchedView.swift
+//  Courses
+//
+//  Created by Dmytro Churakov on 13.10.2022.
+//
+
+import SwiftUI
+
+struct MatchedView: View {
+    @Namespace var namespace
```

Comment:
Maybe it should be NameSpace N and S caps?

Reply:
@Forkipe , it's one word and don't need CamelCase.